lib/al/Library/Screen/ScreenFunction.h

```diff
@@ -0,0 +1,107 @@
+void calcLayoutPosFromWorldPos(sead::Vector2f* output, const al::IUseCamera*, const sead::Vector3f&);
+void calcLayoutPosFromWorldPos(sead::Vector3f* output, const al::IUseCamera*, const sead::Vector3f&);
+void calcLayoutPosFromWorldPos(sead::Vector2f* output, const al::SceneCameraInfo*, const sead::Vector3f&, int);
+void calcLayoutPosFromWorldPos(sead::Vector3f* output, const al::SceneCameraInfo*, const sead::Vector3f&, int);
+void calcLayoutPosFromWorldPosSub(sead::Vector2f* output, const al::IUseCamera*, const sead::Vector3f&);
+void calcLayoutPosFromWorldPosWithClampOutRange(sead::Vector3f* output, const al::IUseCamera*, const sead::Vector3f&, float, int);
+void calcLayoutPosFromWorldPosWithClampOutRange(sead::Vector3f* output, const al::SceneCameraInfo*, const sead::Vector3f&, float, int);
+void calcLayoutPosFromWorldPosWithClampByScreen(sead::Vector3f* output, const al::IUseCamera*, const sead::Vector3f&);
+float calcLayoutRadiusFromWorldRadius(const sead::Vector3f&, const al::IUseCamera*, float);
+
+bool calcCameraPosToWorldPosDirFromScreenPos(sead::Vector3f* output, const al::IUseCamera*, const sead::Vector2f&, float);
+bool calcCameraPosToWorldPosDirFromScreenPos(sead::Vector3f* output, const al::IUseCamera*, const sead::Vector2f&, const sead::Vector3f&);
+bool calcCameraPosToWorldPosDirFromScreenPos(sead::Vector3f* output, const al::SceneCameraInfo*, const sead::Vector2f&, float, int);
+bool calcCameraPosToWorldPosDirFromScreenPos(sead::Vector3f* output, const al::SceneCameraInfo*, const sead::Vector2f&, const sead::Vector3f&, int);
+void calcCameraPosToWorldPosDirFromScreenPosSub(sead::Vector3f* output, const al::IUseCamera*, const sead::Vector2f&, float);
+void calcCameraPosToWorldPosDirFromScreenPosSub(sead::Vector3f* output, const al::IUseCamera*, const sead::Vector2f&, const sead::Vector3f&);
+
+void calcLineCameraToWorldPosFromScreenPos(sead::Vector3f* output1, sead::Vector3f* output2, const al::IUseCamera*, const sead::Vector2f&);
+void calcLineCameraToWorldPosFromScreenPos(sead::Vector3f* output1, sead::Vector3f* output2, const al::IUseCamera*, const sead::Vector2f&, float, float);
+void calcLineCameraToWorldPosFromScreenPos(sead::Vector3f* output1, sead::Vector3f* output2, const al::SceneCameraInfo*, const sead::Vector2f&, int);
+void calcLineCameraToWorldPosFromScreenPos(sead::Vector3f* output1, sead::Vector3f* output2, const al::SceneCameraInfo*, const sead::Vector2f&, float, float, int);
+void calcLineCameraToWorldPosFromScreenPosSub(sead::Vector3f* output1, sead::Vector3f* output2, const al::IUseCamera*, const sead::Vector2f&);
+void calcLineCameraToWorldPosFromScreenPosSub(sead::Vector3f* output1, sead::Vector3f* output2, const al::IUseCamera*, const sead::Vector2f&, float, float);
+} // namespace al
+
+namespace ScreenFunction {
+void calcWorldPositionFromCenterScreen(sead::Vector3f*,sead::Vector2f const&,sead::Vector3f const&,sead::Camera const&,sead::Projection const&,sead::Viewport const&);
```

Comment:
Please run clang format to keep formatting consistent, as this function is missing spaces